### PR TITLE
Radar: added sharing/embed instructions

### PR DESCRIPTION
**Pull request by tiagoad**

### CODE REVIEW

**Comment by haleycode** [comment]
@tiagoad Please remove the screenshot. We try to make sure never to include screenshots unless it's critical for user understanding, due to how difficult it is to maintain screenshots over time.

**Comment by tiagoad** [comment]
> @tiagoad Please remove the screenshot. We try to make sure never to include screenshots unless it's critical for user understanding, due to how difficult it is to maintain screenshots over time.

Understood. It's removed now

File: content/radar/get-started/embed.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+pcx_content_type: reference
+title: Share a Radar chart
+meta:
+    title: Share a Radar chart
+weight: 3
+---
+
+# Share a Radar chart
+
+Radar allows you to download an image of a chart, as well as to embed interactive cards of most charts into your own webpages.
```
Comment (haleycode) [inline content/radar/get-started/embed.md]:
```suggestion
Radar allows you to download an image of a chart, as well as embed interactive cards of most charts into your own web pages.
```

File: content/radar/get-started/embed.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+Charts supporting this feature will have a share icon <img src="/icons/share.svg" style="display: inline-block" title="Share Icon"/> next to its description.
+
+
+## Download a chart in PNG format
+
+1. Press the share button the open the sharing menu
+
+2. Click the button labeled "Download Image"
+
+3. Done! Your browser will download the .png file containing the requested chart
```
Comment (haleycode) [inline content/radar/get-started/embed.md]:
```suggestion
1. Select the Share icon next to the description of the chart you wish to share.
2. Select `Download Image.`
3. A .png file containing the requested chart will be downloaded.
```

File: content/radar/get-started/embed.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+## Embed an interactive chart in your website
+
+1. Press the share button to open the sharing menu
+
+2. Select between **fixed-time** and **real-time**, where:
+   - Real-time uses a “sliding window” based on the selected date range, and will display data points looking back over that duration from the current date/time.
+   - Fixed-time will always display a chart with only the currently visible data points.
+
+3. Copy the HTML to your page
```
Comment (haleycode) [inline content/radar/get-started/embed.md]:
```suggestion
1. Select the Share icon next to the description of the chart you wish to share.
2. Select between Fixed Time and Real Time.
   - Real Time uses a “sliding window” based on the selected date range, and will display data points looking back over that duration from the current date/time.
   - Fixed Time will always display a chart with only the currently visible data points.
3. Select Copy Code and paste the code into your web page.
```

**Review by haleycode — CHANGES_REQUESTED** [review]

**Review by haleycode — APPROVED** [review]

**Pull request merged**